project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=0.13.15
```

Comment:
I don't think we should recommend people using `1.x` just yet. On my projects it's noticeably slower than `0.13.16`.